src/T2DMap.cpp

```diff
@@ -5019,11 +5019,9 @@ void T2DMap::resizeMultiSelectionWidget()
 {
     int newWidth;
     if (mIsSelectionUsingNames) {
-        if (width() <= 300) { // 0 - 300 => 0 - 200
```

Comment:
The original code allowed for a smooth size adjustment based on the overall width of the mapper so that it scaled proportionally over the following ranges:

mapper width | selection widget width if names ARE shown | selection widget width if names are NOT shown
----------------------|-----------------------------------------------------------------------------|---------------------------------------------------------------------
 up to 300 | two thirds of width (so 0 to 200) | two thirds of width (so 0 to 200) and I think I have that slightly wrong so it should only scale up to 120...!
 300 to 600 | one thirds of width + 100 (so 200 to 300) | 120
 600 or more | one thirds of width + 100 (so 300) | 120

The new code just snaps between two fixed widths of 300 - IF THE NAMES ARE SHOWN:

mapper width | selection widget width if names ARE shown | selection widget width if names are NOT shown
----------------------|-----------------------------------------------------------------------------|---------------------------------------------------------------------
 up to 300 | 160 | two thirds of width (so 0 to 200)
 300 to 600 | 160 | 120
 600 to 999 | 160 | 120
 999 or more | 300 | 120

If you *are* going to shrink the width I would urge you to tweak things so there are not snap changes in widths as that is even more annoying (IMHO) in that a slight adjustment in width will produce a sudden change...

Reply:
how often do people adjust the width of their map while having a multiple rooms selected. In my UI I set my container sizes and my mapper never gets its width adjusted.... honestly I felt that 160 pixels was already a little wider than I liked.

It really is a struggle to move rooms while mapping when your mapper is about 600 pixels wide and the popup list is 300 (half of your visible mapper.)

I will try to think of another solution, but I just got done mapping 20+ areas with the new setting of 160 wide and it was sooooooo much easier to get things done.